lnrpc/routerrpc/router_backend.go

```diff
@@ -348,7 +348,10 @@ func (r *RouterBackend) QueryRoutes(ctx context.Context,
 	// Calculate route success probability. Do not rely on a probability
 	// that could have been returned from path finding, because mission
 	// control may have been disabled in the provided ProbabilitySource.
-	successProb := r.getSuccessProbability(route)
+	successProb, err := r.getSuccessProbability(route)
```

Comment:
Longer term, maybe we want to return the success probability directly from `FIndRoute`.